src/sysctl.cr

```diff
@@ -0,0 +1,87 @@
+lib LibC
```

Comment:
Sorry for the misinformation! as this is in `lib_c` you don't need the `skip` macro.